blueocean-rest/src/main/java/io/jenkins/blueocean/rest/pageable/PagedResponse.java

```diff
@@ -28,6 +28,7 @@
 @InterceptorAnnotation(PagedResponse.Processor.class)
 public @interface PagedResponse {
     class Processor extends Interceptor {
+        private static final int DEFAULT_LIMIT=100;
```

Comment:
@tfennelly Not at the moment. We can set it via property if need be but please note that limit query parameter lets you override it as well. What makes it complicated is that with https://issues.jenkins-ci.org/browse/JENKINS-38055, we are going to revisit it to keep per API limit default.